cursorless-talon/src/cursorless_command_server.py

```diff
@@ -0,0 +1,32 @@
+from talon import Module, actions
+
+from .command import Any
+
+mod = Module()
+
+
+@mod.action_class
+class cs_actions:
+    def fs_run_command_with_plugin(action: str, command: Any):
```

Comment:
What does `fs` stand for?  `fail_safe`?  If so, I'd just use `fail_safe`

Also, how come the arg signature doesn't match the one in knausj?

Reply:
Let's just call this one `_cursorless_run_rpc_command`.  Then we can us the same pattern elsewhere in this file, just taking the knausj action name and prefixing `_cursorless`

Reply:
Renamed. I  made the signitures what I saw cursorless required. I've been using this as my cursorless repository, I don't use VS code all that much for work but it seems to work.

Reply:
The signature should mirror the one in knausj

Reply:
Will do.

Reply:
When I made the signatures match I started to get the JSON serialization errors. I know I'm missing something stupid but could you take a look and see what I'm missing?